backend/app/rabbitmq/heartbeat_listener_sync.py

```diff
@@ -1,77 +1,76 @@
+import logging
 import pika
 import json
 from packaging import version
-from app.config import settings
 from pymongo import MongoClient
-from app.models.listeners import (
-    EventListenerDB,
-    EventListenerOut,
-)
+
+from app.config import settings
+from app.models.listeners import EventListenerDB, EventListenerOut, ExtractorInfo
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
 
 
 def callback(ch, method, properties, body):
-    statusBody = json.loads(body.decode("utf-8"))
-    print("received extractor heartbeat: " + str(statusBody))
-    extractor_id = statusBody["id"]
-    extractor_queue = statusBody["queue"]
-    extractor_info = statusBody["extractor_info"]
+    """This method receives messages from RabbitMQ and processes them."""
+    msg = json.loads(body.decode("utf-8"))
+
+    extractor_info = msg["extractor_info"]
     extractor_name = extractor_info["name"]
-    extractor_db = EventListenerDB(**extractor_info)
-    client = MongoClient(settings.MONGODB_URL)
-    db = client["clowder2"]
-    existing_extractor = db["listeners"].find_one({"name": extractor_queue})
+    extractor_parameters = extractor_info["parameters"]["schema"]
+    extractor_info["parameters"] = extractor_parameters
```

Comment:
How come we are reassigning `parameters.schema` to `parameters` in these two lines?

Reply:
this was a mistake. I took these lines out.